tests/contracts/contracts/StakingContractsTestSet.sol

```diff
@@ -287,3 +234,41 @@ contract SimpleStakingContract is AbstractStakingContract, Ownable {
     }
 
 }
+
+
+interface IExtendedStakingEscrow is IStakingEscrow {
+    function deposit(address, uint256, uint16) external;
+    function withdraw(uint256) external;
+}
+
+/**
+* @notice Contract for staking contract tests
+*/
+contract ExtendedStakingInterface is StakingInterface {
+
+    event DepositedAsStaker(address indexed sender, uint256 value, uint16 periods);
+    event WithdrawnAsStaker(address indexed sender, uint256 value);
+
+    constructor(
+        NuCypherToken _token,
+        IStakingEscrow _escrow,
+        PolicyManager _policyManager,
+        WorkLock _workLock
+    )
+        StakingInterface(_token, _escrow, _policyManager, _workLock)
+    {
+    }
+
+    function depositAsStaker(uint256 _value, uint16 _periods) public onlyDelegateCall {
```

Comment:
Are these tests for deposit/withdraw methods that will be used post-merge in some way? How is this related to the staking adaptor contract set?

Reply:
These lines to minimize changes in tests right now